code/game/objects/structures/aliens.dm

```diff
@@ -114,36 +252,53 @@
 	density = FALSE
 	layer = ABOVE_OPEN_TURF_LAYER
 	plane = FLOOR_PLANE
-	icon = 'icons/obj/smooth_structures/alien/weeds1.dmi'
-	icon_state = "weeds1"
-	base_icon_state = "weeds1"
+	icon = 'icons/obj/smooth_structures/alien/weeds.dmi'
+	icon_state = "weeds"
+	base_icon_state = "weeds"
 	max_integrity = 15
+	layer = 2.455
```

Comment:
use the define for this layer please (or create one if it does not exist)

Reply:
👍 Donezo